src/js/components/Ballot/CandidateItem.jsx

```diff
@@ -125,6 +126,16 @@ class CandidateItem extends Component {
   }
 
   render () {
+    const handleHover = () => {
```

Comment:
I would move these functions handleHover and handleLeave out of the render. In the ReactJS world, calling setState directly within the render function is frowned upon.